pkg/config/setup/config_nix.go

```diff
@@ -62,7 +62,7 @@ func osinit() {
 	// Agent binary
 	_here, err := executable.Folder()
 	if err != nil {
-		panic(fmt.Sprintf("Failed to get executable path: %v", err))
+		log.Errorf("Failed to get executable path: %v", err)
```

Comment:
Since entering this scope means that `_here` is invalid, and therefore none of the following variables are going to be assigned meaningless paths, does it make sense to add a return here? I feel like a lot of functionality is going to be subtly broken in this case, but that's probably expected from running the agent in a way that we don't officially support.

Reply:
Yeah that makes total sense to do, added